CBLClient/Apps/CBLTestServer-Dotnet/TestServer.UWP/TestServer.UWP.csproj

```diff
@@ -128,7 +128,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Couchbase.Lite.Enterprise.Support.UWP">
-      <Version>2.1.0-b0223</Version>
+      <Version>2.1.0-b0232</Version>
```

Comment:
is this build version specific? do you need to use 2.1.5 version?

Reply:
can you remove this checkin as it is not required? Once you remove and checkin, you can add these files to gitignore to avoid checking in

Reply:
we should not add these file to git ignore as it leads to unnecessary complications when we open project agains. Build version is not needed and theses files are ignored when the builds are created. When we use project for personal use, we anyways install the build required, which overwrite this no.